### PR TITLE
Refactored JS into Functions

### PR DESCRIPTION
**Pull request by kryolight**

### CODE REVIEW

File: src/solar.js [hunk 1]
```diff
@@ -9,35 +9,51 @@ if ("serviceWorker" in navigator) {
...
 
 /**********
 Create Renderer
 **********/
-var renderer = new THREE.WebGLRenderer({antialias: true});
-renderer.setSize( window.innerWidth, window.innerHeight );
-document.body.appendChild( renderer.domElement );
-
+function createRenderer() {
+  //let renderer = new THREE.WebGLRenderer({antialias: true});
```
Comment (darktears) [inline src/solar.js]: Probably want to remove that.

File: src/solar.js [hunk 2]
```diff
@@ -47,27 +63,29 @@ Create Scene
...
-moonPivot = new THREE.Object3D();
-astronautObj = new THREE.Object3D();
-
-for (var i=0; i < jsonObj.numPlanets; i++){
-  pivots[i] = new THREE.Object3D();
-  pivots[i].position.set(0, 0, 0);
-  scene.add(pivots[i]);
-  planetTargets[i] = new THREE.Object3D();
+function createScene() {
+  //let scene = new THREE.Scene();
```
Comment (darktears) [inline src/solar.js]: Ditto.

File: src/solar.js [hunk 3]
```diff
@@ -518,130 +536,131 @@ var checkRaycasting = (mouse) => {
...
-    this.classList.toggle("active");
-    var content = this.nextElementSibling;
-    if (content.style.display === "block") {
-      content.style.display = "none";
-    } else {
-      content.style.display = "block";
-    }
-  });
-}
+// let coll = document.getElementsByClassName("collapsible");
```
Comment (darktears) [inline src/solar.js]: It's usually not a very good practice to land commented code (especially these big chunks). One can always look at the git history and copy paste back.

File: index.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
 	  </head>
 	  <body>
 
-      <div id="menuOverlay">
+      <!-- <div id="menuOverlay">
```
Comment (darktears) [inline index.html]: Same comment about commented code.

**Comment by asnow4u** [comment]
Irrelevant at this point

**Pull request closed without merging**